### PR TITLE
Test for page.clearMemoryCache()

### PR DESCRIPTION
**Pull request by RavenHursT**

### CODE REVIEW

**Comment by puzrin** [comment]
Please, squash into single commit

**Comment by RavenHursT** [comment]
Odd.. PR still shows multiple commits.. but on my fork, it shows only 1:
https://github.com/CaptoraInc/node-phantom-simple/commits/master

Not sure what else to do here...

File: test/test_page_clear_cache.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            page.open('http://localhost:' + server.address().port, function (err, status) {
+              if (err) {
+                done(err);
+                return;
+              }
+
+              page.clearMemoryCache(function(err){
+                assert.ifError(err);
+                browser.exit(done);
+              });
```
Comment (RavenHursT) [inline test/test_page_clear_cache.js]: @puzrin how's this look?

**Comment by puzrin** [comment]
Let's close it. It doesn't tests feature well anyway.

**Pull request closed without merging**